notebooks/link_analysis/Pagerank_compat.ipynb

```diff
@@ -0,0 +1,871 @@
+    "\n",
+    "\n",
+    "### Some notes about vertex IDs...\n",
+    "* The current version of cuGraph requires that vertex IDs be representable as 32-bit integers, meaning graphs currently can contain at most 2^32 unique vertex IDs. However, this limitation is being actively addressed and a version of cuGraph that accommodates more than 2^32 vertices will be available in the near future.\n",
+    "* cuGraph will automatically renumber graphs to an internal format consisting of a contiguous series of integers starting from 0, and convert back to the original IDs when returning data to the caller. If the vertex IDs of the data are already a contiguous series of integers starting from 0, the auto-renumbering step can be skipped for faster graph creation times.\n",
+    "  * To skip auto-renumbering, set the `renumber` boolean arg to `False` when calling the appropriate graph creation API (eg. `G.from_cudf_edgelist(gdf_r, source='src', destination='dst', renumber=False)`).\n",
+    "  * For more advanced renumbering support, see the examples in `structure/renumber.ipynb` and `structure/renumber-2.ipynb`\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Test Data\n",
+    "We will be using the Zachary Karate club dataset \n",
+    "*W. W. Zachary, An information flow model for conflict and fission in small groups, Journal of\n",
+    "Anthropological Research 33, 452-473 (1977).*\n",
+    "\n",
+    "\n",
+    "![Karate Club](../img/zachary_black_lines.png)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Prep"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Requirement already satisfied: networkx in /home/nfs/dacosta/anaconda3/envs/cugraph_dev/lib/python3.8/site-packages (2.7)\n",
+      "Requirement already satisfied: scipy in /home/nfs/dacosta/anaconda3/envs/cugraph_dev/lib/python3.8/site-packages (1.8.0)\n",
+      "Requirement already satisfied: numpy<1.25.0,>=1.17.3 in /home/nfs/dacosta/anaconda3/envs/cugraph_dev/lib/python3.8/site-packages (from scipy) (1.21.5)\n"
+     ]
+    }
+   ],
+   "source": [
+    "# The notebook compares cuGraph to NetworkX,  \n",
+    "# therefore there some additional non-RAPIDS python libraries need to be installed. \n",
+    "# Please run this cell if you need the additional libraries\n",
+    "!pip install networkx\n",
+    "!pip install scipy"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import needed libraries\n",
+    "from collections import OrderedDict\n",
+    "from scipy.io import mmread"
```

Comment:
I don't see these imports being used anywhere so I think you can remove this entire cell.